client/src/main/com/sinch/sdk/models/E164PhoneNumber.java

```diff
@@ -25,9 +25,11 @@ public class E164PhoneNumber {
 
   private final String number;
 
+  private static boolean strict = true;
```

Comment:
Are you sure you want to active strict mode by default.
And in case the strict mode is set to false, you may print a warning

Reply:
Good point for default value.
For alpha and feedback releases expectation I like the idea to have default to true. But I'm open to change my mind 
Thank you for the message warning: added